code/prime-number/Model/src/test/java/ru/unn/agile/primenumber/model/PrimeNumberFinderTest.java

```diff
@@ -0,0 +1,96 @@
+package ru.unn.agile.primenumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class PrimeNumberFinderTest {
+
+    @Test
+    public void canInitFinder() {
+        var finder = new PrimeNumberFinder(0, 10);
+        assertNotNull(finder);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithNegativeStart() {
+        var finder = new PrimeNumberFinder(-1, 10);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithNegativeEnd() {
+        var finder = new PrimeNumberFinder(0, -1);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithEndBiggerThanStart() {
+        var finder = new PrimeNumberFinder(10, 0);
+    }
+
+    @Test
+    public void cannotFindPrimeNumberBefore2() {
+        var finder = new PrimeNumberFinder(0, 2);
+        var result = finder.findNumbers();
+        assertEquals("", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber1() {
+        var finder = new PrimeNumberFinder(0, 3);
+        var result = finder.findNumbers();
+        assertEquals("2", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber3() {
+        var finder = new PrimeNumberFinder(0, 4);
+        var result = finder.findNumbers();
+        assertEquals("2 3", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber5() {
+        var finder = new PrimeNumberFinder(0, 6);
+        var result = finder.findNumbers();
+        assertEquals("2 3 5", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber7() {
+        var finder = new PrimeNumberFinder(0, 8);
+        var result = finder.findNumbers();
+        assertEquals("2 3 5 7", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber11() {
+        var finder = new PrimeNumberFinder(0, 12);
+        var result = finder.findNumbers();
+        assertEquals("2 3 5 7 11", result);
+    }
+
+    @Test
+    public void canFindPrimeNumber101() {
+        var finder = new PrimeNumberFinder(0, 102);
+        var result = finder.findNumbers();
+        assertTrue(result.contains(" 101"));
+    }
+
+    @Test
+    public void canFindPrimeNumber907() {
+        var finder = new PrimeNumberFinder(0, 1000);
+        var result = finder.findNumbers();
+        assertTrue(result.contains(" 907"));
```

Comment:
Do you need check empty space for checking contained numbers?

Reply:
It's just for check that it is 907 not any number that ends with 907. 
In case of, for example, 1907 in result string it also will be passed. 

If you ask me about why it isn't " 907 " i can say that it is last number of result string of prime numbers, so, there is no white space in the end.

But, for sure, you are prime reviewer, if you want me to remove this white space before 907, I will.

Reply:
I insist on using int - It will be more usefull and safety